src/types_core.ts

```diff
@@ -17,10 +19,12 @@ export class CoreConfig {
   MUTE_TIMEOUT: number
 
   constructor() {
+    if (!process.env.GCP_API_KEY) throw new Error("GCP_API_KEY is missing!")
```

Comment:
Maybe don't make it error? If someone is poor, they should be allowed to use it with rate limiting

Reply:
hahah, nice

Reply:
This is still gonna throw an error btw

Reply:
fixed